src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/InProcess/RoslynSettingsFileSynchronizerTests.cs

```diff
@@ -184,21 +184,21 @@ private static BindingConfiguration CreateConnectedConfiguration(string projectK
             return BindingConfiguration.CreateBoundConfiguration(project, SonarLintMode.Connected, "some directory");
         }
 
-        private SuppressedIssuesFileSynchronizer CreateTestSubject(ISuppressedIssuesMonitor suppressedIssuesMonitor = null,
+        private RoslynSettingsFileSynchronizer CreateTestSubject(ISuppressedIssuesMonitor suppressedIssuesMonitor = null,
             ISonarQubeIssuesProvider suppressedIssuesProvider = null,
-            ISuppressedIssuesFileStorage suppressedIssuesFileStorage = null,
+            IRoslynSettingsFileStorage suppressedIssuesFileStorage = null,
```

Comment:
variable is still the old name but it's not a big deal, we can keep it

Reply:
I'll fix up the variable names.